content/blog/open-sourcing-workshops-on-demand-part3-understanding-the-backend.md

```diff
@@ -0,0 +1,340 @@
+
+I﻿n terms of task associated to the **PURGE** scenario, you can see that we kept the minimal as there should not be much to clean up on the backend server. Same tasks to begin with as we still need student id and workshop id. 
+
+W﻿e then initiate :
+
+9- `generate_randompwd()`:  We always update the sdtudent's password for security reasons.
+
+1﻿0- `erase-student()`:  we perform a cleanup of the student folder.
+
+1﻿1- API calls to to update tables in the database. The new s﻿tudent password is recorded. We also generate a new password at the **PURGE** phase to prevent unregistered logins. The s﻿tudent status is set to inactive. The capacity figure is incremented by one to make the seat available again. 
+
+A﻿n email is then sent to the participant explaining to him that we encountered an issue with the deployment and that we apologize for this. The same email is sent to the admin so he can work on the issue.
+
+N﻿ow, you should have a clearer view of what is really happening in the background when one registers for a workshop. You can see that I have uncovered many scripts to explain step by step all the stages of a workshop's deployment process. But there are some more to be explained. It is obvious that the main function of the backend server is to deploy and run workshops. Nevertheless, as any other server, it cannot live without maintenance.
+
+T﻿his subject will be at the core of my next article where I will detail how one needs to manage and work with this server on a daily basis. What we usually call day 2 operations.
+
+Please be sure to drop back at [HPE DEV](https://developer.hpe.com/blog) for a follow up on this. Check out also the Hack Shack for new [workshops](https://developer.hpe.com/hackshack/workshops)! We are currently working on different subjects like HPE GreenLake for Compute Operations Management API 101 or HPE Data Services Cloud Console API for HPE GreenLake for Block Storage cloud data service. Stay Tuned !
```

Comment:
Please see my comments from the previous blog post on your summary. We should spell out HPE Developer blog as opposed to just HPE DEV and it would probably be a good idea to add in a Slack link with a statement like "If we can be of any help in clarifying any of this, please reach out to us on Slack."